spec/spec_helper.rb

```diff
@@ -30,7 +33,7 @@
     #     # => "be bigger than 2"
     expectations.include_chain_clauses_in_custom_matcher_descriptions = true
   end
-
+  
```

Comment:
Adding a bunch of carriage returns makes for a possible merge conflict for all of your teammates on those lines. While I agree it may make things look better, it is bad practice to change the diff without an explicit reason.